src/components/snapshots/useSnapshot.ts

```diff
@@ -68,9 +68,16 @@ const useSnapshot: SnapshotOperation = async ({
       // The quick way, using pg_restore (whole database only, no partial
       // exports)
       console.log('Restoring database...')
+
+      // Safer to drop and recreate whole schema, as there can be errors when
+      // trying to drop individual objects using --clean, especially if the
+      // incoming database differs from the current database, schema-wise
+      execSync(`psql -U postgres -d tmf_app_manager -c 'DROP schema public CASCADE;'`)
+      execSync(`psql -U postgres -d tmf_app_manager -c 'CREATE schema public;'`)
```

Comment:
Okay, this seems to fix #925 as far as I can tell.

Please test by loading different snapshots (core/fiji/angola) in various orders and check that the restored database is correct.

Reply:
Ok - Hopefully it's fixed. I'm testing now

Reply:
Have swap between these snapshots and not seen any problems